### ISSUE TITLE
Steam beta breaks HTC Vive in SteamVR

### ISSUE THREAD

**Issue by celphy** [ValveSoftware/steam-for-linux#8112]
#### Your system information

* Steam client version (build number or date): Sep 17 2021, at 04:30:38
* Distribution (e.g. Ubuntu): Fedora 34 (5.13.19-200.fc34.x86_64
* Opted into Steam client beta?: Yes
* Have you checked for system updates?: Yes

#### Please describe your issue in as much detail as possible:
I expected SteamVR to pick up my HTC Vive after opting into the Steam beta branch. Instead it doesn't recognize my HTC Vive as a headset at all. This is fixed by opting out of the beta. 

As this somehow doesn't see SteamVR related enough I put it here, I hope this is correct. Thanks.

#### Steps for reproducing this issue:

1. Verify SteamVR works with HTC Vive
2. Opt into Steam beta branch
3. Verify SteamVR doesn't work with HTC Vive

**Comment by kisak-valve** [comment]
Hello @celphy, in a quick activation test on my Gentoo VR test box, the September 29 build of Steam from the Steam client beta update channel and both SteamVR 1.19.7 and 1.20.2 initialized a Vive as expected, so at least SteamVR is not unconditionally broken.

If you switch to Steam's beta client, then completely close Steam, and run `steam` from a terminal, are there any hints in the terminal spew when starting SteamVR? Also, it might be interesting to compare SteamVR's system report (https://github.com/ValveSoftware/SteamVR-for-Linux/blob/master/CapturingSystemReport.md) between working and broken runs and see if there are any hints.

**Comment by celphy** [comment]
So I tried replicating the issue for the last hour and am not able to successfully do so. There were no software updates that stick out to me. According to dfns history these were updated: https://gist.github.com/celphy/ee9529321049151c870856d62e4eded8

Even though I was able to replicate the issue consistently yesterday with both mini-DP and HDMI cables and after rebooting it seems to simply have disappeared. I'll put it down as a hardware fluke and apologize for the inconvenience.

Thanks :)
